src/DisplayBoard.cpp

```diff
@@ -51,7 +51,7 @@ std::string display(const Board& b)
             output << drawRow(widthRows, {});
         else
             output << drawRow(widthRows, cluesRows.at(i));
-        output << drawEmptyLine(width, '?') << "\n";
+        output << drawBoardLine(width, const_cast<Board&>(b).getBoardLines().at(i)) << "\n";
```

Comment:
Probably when you pass a `BLine const &` then you will not need a `const_cast` here. But I may be wrong. Compiler will tell you the truth :)

Reply:
getBoardLines() return  BLines& - and unfortunately do not return a const object, that is why I use in this line const_cast.
 In other tasks, to solving algorithm there will be used getBoardLines() to get value and changed it. 
I really don't know how in another way get board_ as a BLine & const - and I don't think that writing the second method will be a good solution...so, I don't know.

Reply:
Take a look at my changes: https://github.com/LordLukin/LogicalImages/pull/51/commits/6f3d0aba23ae3b5e031ded28843e2c41b70d4c6b
There were some missing consts in `getBoardLines()` function. Now it works without const_cast :)